prisma/schema.prisma

```diff
@@ -138,6 +138,7 @@ model Reference {
   hasKnowApplicantForAYear Boolean?
   isRelated                Boolean?
   isMentorRecommended      Boolean?
+  isRejected               Boolean?
```

Comment:
Hi Tristan, the table has a `isMentorRecommended` column. What about leveraging that column?

Reply:
Sounds good, I have a few ideas. Will play around with some things.